### ISSUE TITLE
Accordion heading links have no href

### ISSUE THREAD

**Issue by thomassuckow** [angular-ui/bootstrap#2226]
The heading links for the accordion have no href making them not accessible to keyboard navigation.

**Comment by tmundt** [comment]
Is it possible to determine a href-Link to an "accordion-group heading"? I would like to show another partial in an different part of the page if clicking on a heading of a accordion.
